apps/client/src/components/RegisterLookupSummary.js

```diff
@@ -43,31 +43,34 @@ const RegisterLookupSummary = ({
 
   return (
     <ComponentWrapper marginBottom={sttrFile ? "0" : null}>
-      <Paragraph gutterBottom={16}>
-        {compact ? (
+      {compact ? (
+        <AddressLine {...address} />
+      ) : (
+        <>
+          <Paragraph strong gutterBottom={0}>
+            Ingevoerd adres:
+          </Paragraph>
```

Comment:
This looks better and easier to understand:
```
{!compact && <Paragraph strong gutterBottom={0}>Ingevoerd adres:</Paragraph>}

<AddressLine {...address} />

{!compact && <ChangeAddressModal {...{ hasSTTR, setActiveState, slug }} />}
```

Reply:
This is fixed in an other PR